config/routes.rb

```diff
@@ -24,6 +24,9 @@
   resources :holidays, only: %i(index new create) do
     delete '/', on: :collection, action: :destroy
   end
+  resources :bookable_slots, only: [] do
```

Comment:
`get '/bookable_slots/available', to: 'bookable_slots#available'` makes more sense here I think.